requirements.txt

```diff
@@ -53,7 +53,8 @@ titlecase==1.1.1
 social-auth-app-django==4.0.0
 Python-jose==3.2.0
 gitpython>=2.1.13
-ptvsd==4.3.2
+ptvsd>=4.3.2
```

Comment:
Could you fix a version ? not fixing version will break something in the futur

Reply:
dependabot should keep us up to date.

Reply:
I agree dependabot is a more reliable system. it test **only** one upgrade and create the PR